### PR TITLE
Tweaks for Universe Tests

### PR DESCRIPTION
**Pull request by pranavkm**
* Use --quiet for git pull and git clone so git fetch messages do not show up in CI error logs.
* Use --parallel for default verify target.

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @victorhurdugaci \ @dougbu

**Comment by victorhurdugaci** [comment]
Looks good :shipit:

File: makefile.shade [hunk 1]
```diff
@@ -82,7 +82,7 @@ var buildTarget = "compile"
 
 #change-default-build-target
     @{
-        buildTarget = "verify";
+        buildTarget = "--parallel verify";
```
Comment (dougbu) [inline makefile.shade]: Need a way to disable `--parallel` if enabling it in this fashion. This also isn't a great default due to the spew of messages coming from the different threads -- makes finding a problem more difficult. (Could change the `Parallel.ForEach()` blocks to spew with `msbuild`-like thread ids but that's probably too much work for this private build system.)

**Comment by dougbu** [comment]
:shipit: after adding `--noparallel` or similar.

File: makefile.shade (hunk 1 above)
Comment (pranavkm) [inline makefile.shade]: Will remove this for now. Just doing the `--quiet` to reduce CI spew.

**Pull request closed without merging**